CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change log
 -----------
 
+# v2.0.4+rev2 - 2017-05-28
```

Comment:
I think even in the past we did a release in every meta-resin bump but we included the modifications needed for that release. Anyway - it's more of a suggestion.